test/langtools/tools/javac/lambda/lambdaExpression/LambdaTest6.java

```diff
@@ -26,14 +26,16 @@
  * @bug 8003280
  * @summary Add lambda tests
  *   Test bridge methods for certain SAM conversions
- *   Tests that jdk.internal.lambda.disableEagerInitialization=true creates a
- *   get$Lambda method for non-capturing lambdas
+ *   Test the set of generate fields
```

Comment:
I would suggest to consider having the test under test/jdk/(java/lang/invoke/lambda), not under test/langtools/tools/javac.

Reply:
It's a good suggestion as `disableEagerInitialization` support is not part of javac.

Reply:
OK makes sense. I guess it's still good to clean the test comments of the old `disableEagerInitialization` references?

Reply:
I have created a new test under `test/jdk/java/lang/invoke/lambda` and cleaned up `LambdaTest6.java`